unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/FriendsController.cs

```diff
@@ -131,16 +193,27 @@ public async UniTask<FriendRequest> RejectFriendshipAsync(string friendRequestId
         {
             cancellationToken.ThrowIfCancellationRequested();
 
-            RejectFriendshipPayload payload = await apiBridge.RejectFriendshipAsync(friendRequestId, cancellationToken);
+            FriendRequest friendRequest;
 
-            FriendRequestPayload requestPayload = payload.FriendRequestPayload;
-            var request = ToFriendRequest(requestPayload);
-            friendRequests.Remove(request.FriendRequestId);
+            if (useSocialApiBridge)
```

Comment:
We should implement a proxy pattern to avoid having this two implementations coexisting here. It would be another implementation of `IRPCSocialApiBridge` that wrappes the usage of the new or old flow. 

You can find a good example here: `WearablesCatalogServiceProxy`